src/pdf/ShoppingList/ShoppingListPDF.tsx

```diff
@@ -0,0 +1,224 @@
+"use client";
+
+import React from "react";
+import { Document, Page, Text, View, Image, StyleSheet } from "@react-pdf/renderer";
+import {
+    BlockProps,
+    ClientSummary,
+    Item,
+    ShoppingListPDFProps,
+} from "@/pdf/ShoppingList/dataPreparation";
+
+const styles = StyleSheet.create({
+    paper: {
+        margin: "0.75in",
+        lineHeight: "1.5pt",
+    },
+    flexRow: {
+        display: "flex",
+        flexDirection: "row",
+    },
+    flexColumn: {
+        display: "flex",
+        flexDirection: "column",
+    },
+    pdfHeader: {
+        justifyContent: "space-between",
+    },
+    logoStyling: {
+        maxWidth: "20%",
+        alignSelf: "center",
+    },
+    infoBlock: {
+        borderStyle: "solid",
+        border: "1pt",
+    },
+    infoCell: {
+        width: "50%",
+        padding: "5pt",
+    },
+    table: {
+        borderStyle: "solid",
+        border: "1pt",
+        padding: "5pt",
+    },
+    tableRow: {
+        borderBottomStyle: "solid",
+        borderBottom: "1pt",
+        paddingVertical: "4pt",
+        textAlign: "center",
+    },
+    tableItemDescription: {
+        width: "25%",
+    },
+    tableQuantity: {
+        width: "35%",
+    },
+    tableNotes: {
+        width: "30%",
+    },
+    tableDone: {
+        width: "10%",
+    },
+    title: {
+        fontSize: "30pt",
+        fontFamily: "Helvetica-Bold",
+    },
+    subtitle: {
+        fontSize: "24pt",
+        fontFamily: "Helvetica",
+    },
+    keyText: {
+        fontSize: "12pt",
+        fontFamily: "Helvetica-Bold",
+    },
+    normalText: {
+        fontSize: "12pt",
+        fontFamily: "Helvetica",
```

Comment:
I ain't sure but seems like for my side if I didn't set any font it will be by default this font so maybe it is not necessary?